mmgen/models/diffusions/utils.py

```diff
@@ -0,0 +1,245 @@
+import torch
+
+
+def _get_noise_batch(noise,
+                     image_shape,
+                     num_timesteps=0,
+                     num_batches=0,
+                     timesteps_noise=False):
+    """Get noise batch. Support get sequeue of noise along timesteps.
+    If passed noise is a torch.Tensor,
```

Comment:
Maybe you can move this comments to the corresponding code lines below. This comments do not make me understand your intentions of expanding/ reshaping.

Reply:
Also noticing other places following this commenting style.

Reply:
In-line comments are added as well.